packages/adminrouter/extra/src/lib/cache.lua

```diff
@@ -11,20 +11,51 @@ local util = require "util"
 --
 -- CACHE_FIRST_POLL_DELAY << CACHE_EXPIRATION < CACHE_POLL_PERIOD < CACHE_MAX_AGE_SOFT_LIMIT < CACHE_MAX_AGE_HARD_LIMIT
 --
--- CACHE_BACKEND_REQUEST_TIMEOUT << CACHE_REFRESH_LOCK_TIMEOUT
 --
+-- Before changing CACHE_POLL_PERIOD, please check the comment for resolver
```

Comment:
I expect this is where the conflict occurred. This line should be removed. The full sentence appears in lines 28-29.

Reply:
We discussed that we'd like to see resolution of this tracked via an individual JIRA ticket so that we can move forward with this train for the next load test.